docker-compose.yml

```diff
@@ -228,7 +228,7 @@ services:
 
   mysql-5-6:
     image: mysql:5.6.50
-    command: mysqld
+    command: mysqld --table_definition_cache=2000
```

Comment:
What is the reason for this in this PR?

Reply:
Same as above, too many tests broke MySQL 5.6

Reply:
This was fixed in MySQL 5.7, only 5.1 to 5.6 had this issue.